### ISSUE TITLE
Uncaught TypeError: Cannot read property 'length' of undefined

### ISSUE THREAD

**Issue by danosier** [jakesankey/line-diff-details#27]
I was able to reproduce the error first mentioned in #18. Removed second line of file, when clicking on line 1 to see changes popup, I get this error. 
Atom 1.9.5
line-diff-details 1.3.5

**Comment by kuzyn** [comment]
This happens for me when I'm trying to bring up a `line-diff-detail` contextual menu on a folded line (https://atom.io/packages/line-diff-details)

![screen shot 2016-08-17 at 12 44 07 pm](https://cloud.githubusercontent.com/assets/1139068/17745122/a725c0da-6478-11e6-90d7-0ed493e24283.png)

![screen shot 2016-08-17 at 12 44 16 pm](https://cloud.githubusercontent.com/assets/1139068/17745121/a723096c-6478-11e6-9f3a-bfc856290084.png)

**Comment by marcusorjames** [comment]
+1

**Comment by ChalkPE** [comment]
```
TypeError: Cannot read property 'length' of undefined
    at LineDiffWorker.decorateDiffMarkers (/home/chalk/.atom/packages/line-diff-details/lib/line-diff-worker.coffee:43:67)
    at HTMLDivElement.<anonymous> (/home/chalk/.atom/packages/line-diff-details/lib/line-diff-worker.coffee:33:14)
    at HTMLDivElement.jQuery.event.dispatch (/home/chalk/.atom/packages/line-diff-details/node_modules/jquery/dist/jquery.js:4435:9)
    at HTMLDivElement.elemData.handle (/home/chalk/.atom/packages/line-diff-details/node_modules/jquery/dist/jquery.js:4121:28)
```

**Comment by tomwganem** [comment]
+1

**Comment by jakesankey** [comment]
This should be fixed with 1.8.0
